File: ios/Khrysalis Template/com/lightningkite/khrysalistemplate/api/APIInterface.shared.swift
//Package: com.lightningkite.khrysalistemplate.api
//Converted using Khrysalis2

import Foundation
import Khrysalis
import RxSwift
import RxRelay
import KhrysalisMaps
import KhrysalisBluetooth



public protocol APIInterface {
    
    func getExamplePosts(onResult: @escaping (Int32, Array<Post>?, String?) -> Void) -> Void
}
 

File: android/src/main/java/com/lightningkite/khrysalistemplate/api/APIInterface.shared.kt
package com.lightningkite.khrysalistemplate.api

import com.lightningkite.khrysalis.escaping
import com.lightningkite.khrysalistemplate.models.Post
import io.reactivex.Single

interface APIInterface {
    //https://jsonplaceholder.typicode.com/posts
    fun getExamplePosts(): Single<List<Post>>
}
